util/proposal.go

```diff
@@ -34,5 +40,23 @@ func GetProposalName(payload string) (string, error) {
 
 // IsValidProposalName reports whether str is a valid proposal name
 func IsValidProposalName(str string) bool {
+	regexMounted := MountValidRegExpProposalName()
+	validProposalName := regexp.MustCompile(regexMounted)
 	return validProposalName.MatchString(str)
 }
+
+//MountValidRegExpProposalName return a regex string for matching the proposal name
```

Comment:
Nitpick: there should be a space between `//` and the first character of the comment.